src/app/components/ButtonNavigate/ButtonNavigate.tsx

```diff
@@ -0,0 +1,17 @@
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const ButtonNavigate = styled(Link)`
```

Comment:
Not sure about the naming here. This might look like a button but it's actually a link. I'd consider renaming this component :)